### PR TITLE
Dit 2809 - changes to complete case confirmation pages and "are you sure" atar page

### PR DESCRIPTION
**Pull request by oliwiaKot**
needs this PR to be merged in first https://github.com/hmrc/tariff-classification-frontend/pull/524
covers tickets:
https://jira.tools.tax.service.gov.uk/browse/DIT-2808
https://jira.tools.tax.service.gov.uk/browse/DIT-2809
https://jira.tools.tax.service.gov.uk/browse/DIT-2817
https://jira.tools.tax.service.gov.uk/browse/DIT-2825

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -507,7 +508,7 @@ page.title.ruling.not.found=Ruling not found
 page.title.case.queues=Tariff classification case teams
 page.title.create_liability.h1=New liability information
 page.title.confirmation.h1=New case information
-page.title.create_correspondence.h1=Create correspondence case
+page.title.create_correspondence.h1=Correspondence case released to this team
```
Comment (vajarova) [inline conf/messages:511]: Hiya @oliwiaKot 
I think this title should be split as two different - one is used in the create correspondence journey, and the other one is matching the completed/release page where the h1 is this case has been created
I think best to discuss it with @shiromi  ?

File: app/assets/stylesheets/layout.scss [hunk 2]
```diff
@@ -590,4 +590,22 @@ table tbody tr th {
...
+   line-height: 1.04167;
+}
+.govuk-panel__title {
+    margin-top: 0;
+    margin-bottom: 30px;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 700;
+    font-size: 32px;
+    /* font-size: 2rem; */
```
Comment (emilia-trendafilova) [inline app/assets/stylesheets/layout.scss:609]: remove if not needed

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -590,4 +590,22 @@ table tbody tr th {
 
 .word-break {
     word-break: break-word;
+}
+.govuk-panel__body {
+   -webkit-font-smoothing: antialiased;
+   -moz-osx-font-smoothing: grayscale;
+   font-weight: 400;
+   font-size: 24px;
+   //font-size: 1.5rem;
```
Comment (emilia-trendafilova) [inline app/assets/stylesheets/layout.scss:599]: remove if not needed

File: app/assets/stylesheets/layout.scss (hunk 2 above)
Comment (oliwiaKot) [reply app/assets/stylesheets/layout.scss]: done

File: app/assets/stylesheets/layout.scss (hunk 3 above)
Comment (oliwiaKot) [reply app/assets/stylesheets/layout.scss]: done

File: conf/messages (hunk 1 above)
Comment (oliwiaKot) [reply conf/messages]: I pulled changes from https://github.com/hmrc/tariff-classification-frontend/pull/528/commits/00ef945b16bac548556437f2fd305e695656eeda like @shiro advised and also reviewed the titles on the ages I worked on and corrected them, hope this is okay now @vajarova

**Review by vajarova — APPROVED** [review]

**Pull request merged**